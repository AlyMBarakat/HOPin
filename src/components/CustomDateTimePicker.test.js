import React from 'react';
import renderer from 'react-test-renderer';
import DateTimePicker from './CustomDataTimePicker';

test('renders date and time pickers correctly', () => {
  const tree = renderer
    .create(
      <>
        <DateTimePicker name="Date" type="date" />
        <DateTimePicker name="Time" type="time" />
      </>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
