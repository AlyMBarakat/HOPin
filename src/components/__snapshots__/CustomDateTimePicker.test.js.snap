// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DateTimePicker correctly 1`] = `
Array [
  <View
    style={
      Object {
        "marginBottom": 22,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "flex-start",
          "color": "#fff",
          "fontSize": 15,
          "fontWeight": "bold",
          "marginBottom": 12,
        }
      }
    >
      Date
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "flex-start",
          "backgroundColor": "#72B5A8",
          "borderColor": "#3C796D",
          "borderRadius": 5,
          "borderWidth": 3,
          "opacity": 1,
          "paddingHorizontal": 6,
          "paddingVertical": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2D3436",
          }
        }
      >
        2020-08-22
      </Text>
    </View>
  </View>,
  <View
    style={
      Object {
        "marginBottom": 22,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "flex-start",
          "color": "#fff",
          "fontSize": 15,
          "fontWeight": "bold",
          "marginBottom": 12,
        }
      }
    >
      Time
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "flex-start",
          "backgroundColor": "#72B5A8",
          "borderColor": "#3C796D",
          "borderRadius": 5,
          "borderWidth": 3,
          "opacity": 1,
          "paddingHorizontal": 6,
          "paddingVertical": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2D3436",
          }
        }
      >
        1:15 AM
      </Text>
    </View>
  </View>,
]
`;

exports[`renders date and time pickers correctly 1`] = `
Array [
  <View
    style={
      Object {
        "marginBottom": 22,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "flex-start",
          "color": "#fff",
          "fontSize": 15,
          "fontWeight": "bold",
          "marginBottom": 12,
        }
      }
    >
      Date
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "flex-start",
          "backgroundColor": "#72B5A8",
          "borderColor": "#3C796D",
          "borderRadius": 5,
          "borderWidth": 3,
          "opacity": 1,
          "paddingHorizontal": 6,
          "paddingVertical": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2D3436",
          }
        }
      >
        2020-08-22
      </Text>
    </View>
  </View>,
  <View
    style={
      Object {
        "marginBottom": 22,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "flex-start",
          "color": "#fff",
          "fontSize": 15,
          "fontWeight": "bold",
          "marginBottom": 12,
        }
      }
    >
      Time
    </Text>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "flex-start",
          "backgroundColor": "#72B5A8",
          "borderColor": "#3C796D",
          "borderRadius": 5,
          "borderWidth": 3,
          "opacity": 1,
          "paddingHorizontal": 6,
          "paddingVertical": 10,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2D3436",
          }
        }
      >
        1:15 AM
      </Text>
    </View>
  </View>,
]
`;
